web/packages/teleterm/src/ui/Search/useSearch.test.tsx

```diff
@@ -268,6 +271,49 @@ describe('useResourceSearch', () => {
     await result.current('foo', [], true);
     expect(appContext.resourcesService.searchResources).not.toHaveBeenCalled();
   });
+
+  it('requestable resources should be fetched for leaves if the root cluster allows it', async () => {
```

Comment:
What if the leaf doesn't support it? Does it mean that the request will be simply "downgraded" to include only accessible resources?

Reply:
You mean a situation where the leaf has `ShowResources.ACCESSIBLE_ONLY` set?
This setting is read only be the end clients (web UI and Connect), so it would be ignored. But tbh, there is no way to support it, we read `/web/config.js` only for the root. Also, the docs say `# Configuration options for the Web UI served by the Proxy Service.`, so it should be clear that it's configurable only for the root.

Reply:
I mean a situation where the root is on a version that supports `includeRequestable` and the leaf is on a version which doesn't know about this field.

Reply:
Ah yes, the old version will just ignore this field.